MyBlog.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Areas.Users;
+
+namespace MyBlog.Controllers;
+
+[ApiController]
+[Route("[controller]")]
```

Comment:
in the task, i asked you to create next endpoint: api/users
so, your base route should be: api/users

Reply:
also, you miss to write tag, that describes type, that your controller produce:
[Produces(MediaTypeNames.Application.Json)]